packages/embed/e2e/applitools-utils.js

```diff
@@ -0,0 +1,22 @@
+import { screenSizeDesktop, screenSizeMobile } from '../../../e2e/cypress-utils'
```

Comment:
This import is not resolving correctly for me. It should be `./cypress-utils` right?